src/main/java/com/candyshop/islodycze/registration/CustomUserDetails.java

```diff
@@ -0,0 +1,56 @@
+package com.candyshop.islodycze.registration;
+
+import java.util.Collection;
+
+import com.candyshop.islodycze.model.User;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+public class CustomUserDetails implements UserDetails {
+
+    private User user;
+
```

Comment:
Do we need all functions like isCredentialsNonExpired?

Reply:
Implementation of these methods is forced by interface, they have to stay